### PR TITLE
chore: add v1-alpha registration for backwards compatibility

### PR DESCRIPTION
**Pull request by Abhinandan-Purkait**
Signed-off-by: Abhinandan Purkait <purkaitabhinandan@gmail.com>

### CODE REVIEW

File: rpc/src/lib.rs [hunk 1]
```diff
@@ -138,3 +138,9 @@ pub mod v1 {
         };
     }
 }
+
+pub mod v1_alpha {
```
Comment (tiagolobocastro) [inline rpc/src/lib.rs:143]: add doc comment explaining what v1_alpha?

File: control-plane/agents/src/bin/core/node/mod.rs [hunk 2]
```diff
@@ -22,7 +22,8 @@ pub(crate) async fn configure(builder: agents::ServiceEmpty) -> Service {
 
     builder
         .with_service(node_grpc_service.into_grpc_server())
-        .with_service(registration_service.into_grpc_server())
+        .with_service(registration_service.clone().into_v1_grpc_server())
+        .with_service(registration_service.into_v1_alpha_grpc_server())
```
Comment (tiagolobocastro) [inline control-plane/agents/src/bin/core/node/mod.rs:26]: nice :+1:

Comment (tiagolobocastro) [inline control-plane/agents/src/bin/core/node/mod.rs:26]: nice :+1:

**Review by tiagolobocastro — APPROVED** [review]

File: rpc/src/lib.rs (hunk 1 above)
Comment (Abhinandan-Purkait) [reply rpc/src/lib.rs:143]: Done.

**Comment by Abhinandan-Purkait** [comment]
bors try

**Comment by Abhinandan-Purkait** [comment]
bors merge

**Pull request merged**